frontend/src/metabase/visualizations/components/settings/ChartSettingLinkUrlInput.unit.spec.tsx

```diff
@@ -80,7 +80,7 @@ describe("ChartSettingLinkUrlInput", () => {
     const options = await getOptions();
 
     userEvent.click(options[1]);
-    input.blur();
+    userEvent.tab();
```

Comment:
will be reverted before merge. I used it only for triggering unit tests

Reply:
@uladzimirdev  Should we add `.nvmrc` to [`frontend_sources`](https://github.com/metabase/metabase/blob/master/.github/file-paths.yaml#L18)?